### ISSUE TITLE
Unrecognized metadata value?

### ISSUE THREAD

**Issue by renedupont** [urbica/martin#799]
Hey,

I receive a geojson file that I use to create 5 layers (each in its own mbtiles file) with [tippecanoe](https://github.com/felt/tippecanoe). 

Once I start martin with these mbtiles as source, I get the following warnings:
```
[2023-08-09T09:40:07Z INFO martin::file_config] Configured source level_1 from /mbtiles/level_1.mbtiles
[2023-08-09T09:40:07Z WARN martin_mbtiles::mbtiles] level_1 has an unrecognized metadata value antimeridian_adjusted_bounds=-10.621100,35.004200,40.495800,59.995800
[2023-08-09T09:40:07Z WARN martin_mbtiles::mbtiles] level_1 has an unrecognized metadata value generator_options=tippecanoe '--output=mbtiles/level_1.mbtiles' /geojsons/levels_20230703.geojson '--minimum-zoom=6' '--maximum-zoom=11' '--layer=level1' '--feature-filter={ "*": [ "==", "id", 1 ] }'
[2023-08-09T09:40:07Z WARN martin_mbtiles::mbtiles] level_1 has an unrecognized metadata value strategies=[{},{},{},{},{},{},{"tiny_polygons":5},{"tiny_polygons":14},{"tiny_polygons":39},{"tiny_polygons":134},{"tiny_polygons":321},{"tiny_polygons":722}]
[2023-08-09T09:40:07Z INFO martin::file_config] Configured source level_2 from /mbtiles/level_2 .mbtiles
[2023-08-09T09:40:07Z WARN martin_mbtiles::mbtiles] level_2 has an unrecognized metadata value antimeridian_adjusted_bounds=-10.621100,35.004200,40.495800,59.995800
[2023-08-09T09:40:07Z WARN martin_mbtiles::mbtiles] level_2 has an unrecognized metadata value generator_options=tippecanoe '--output=mbtiles/level_2 .mbtiles' /geojsons/levels_20230703.geojson '--minimum-zoom=6' '--maximum-zoom=11' '--layer=level2' '--feature-filter={ "*": [ "==", "id", 2 ] }'
[2023-08-09T09:40:07Z WARN martin_mbtiles::mbtiles] level_2 has an unrecognized metadata value strategies=[{},{},{},{},{},{},{"tiny_polygons":8},{"tiny_polygons":11},{"tiny_polygons":42},{"tiny_polygons":137},{"tiny_polygons":362},{"tiny_polygons":1024}]
...
```

My question would be: Is that just some custom metadata fields that tippecanoe adds to the mbtiles that martin doesn't care about because it doesn't know what it is and I can fully neglect the warnings, or does this have any consequences during runtime?

**Comment by sharkAndshark** [comment]
Hi @renedupont  Have a good day  :D  
I think you can ignore these warnings .
Based on the [MBTiles spec](https://github.com/mapbox/mbtiles-spec/blob/master/1.3/spec.md#metadata), there are many meta data vallues like:  name, format, bounds, center, minzoom, maxzoom etc. 
Martin will show a warning if there are metadatas not in the spec.
```rust
        while let Some(row) = rows.try_next().await? {
            if let (Some(name), Some(value)) = (row.name, row.value) {
                match name.as_ref() {
                    "name" => tj.name = Some(value),
                    "version" => tj.version = Some(value),
                    "bounds" => tj.bounds = self.to_val(Bounds::from_str(value.as_str()), &name),
                    "center" => tj.center = self.to_val(Center::from_str(value.as_str()), &name),
                    "minzoom" => tj.minzoom = self.to_val(value.parse(), &name),
                    "maxzoom" => tj.maxzoom = self.to_val(value.parse(), &name),
                    "description" => tj.description = Some(value),
                    "attribution" => tj.attribution = Some(value),
                    "type" => layer_type = Some(value),
                    "legend" => tj.legend = Some(value),
                    "template" => tj.template = Some(value),
                    "json" => json = self.to_val(serde_json::from_str(&value), &name),
                    "format" | "generator" => {
                        tj.other.insert(name, Value::String(value));
                    }
                    _ => {
                        let file = &self.filename;
                        warn!("{file} has an unrecognized metadata value {name}={value}");
                        tj.other.insert(name, Value::String(value));
                    }
                }
            }
        }
```

**Comment by nyurik** [comment]
I wonder if we should lower the priority for the unrecognized mbtiles messages to `info` ?

**Comment by sharkAndshark** [comment]
I agree with you. It's more like info @nyurik

**Comment by nyurik** [comment]
PRs welcome :)

**Comment by renedupont** [comment]
Got it, thanks a lot @sharkAndshark !
Closing due to merged PR #800 .
